plugins/CuraEngineBackend/StartSliceJob.py

```diff
@@ -216,12 +216,11 @@ def run(self) -> None:
                 if temp_list:
                     object_groups.append(temp_list)
 
-            extruders_enabled = {position: stack.isEnabled for position, stack in CuraApplication.getInstance().getGlobalContainerStack().extruders.items()}
+            extruders_enabled = {position: stack.isEnabled for position, extruder in stack.extruders.items()}
```

Comment:
When slicing:

Traceback (most recent call last):
2018-07-06 10:40:14,495 - ERROR - [Thread-3] UM.Logger.logException [88]:   File "/Users/m.suurmond/Develop/Uranium/UM/JobQueue.py", line 121, in run
2018-07-06 10:40:14,495 - ERROR - [Thread-3] UM.Logger.logException [88]:     job.run()
2018-07-06 10:40:14,496 - ERROR - [Thread-3] UM.Logger.logException [88]:   File "/Users/m.suurmond/Develop/Cura/cura/../plugins/CuraEngineBackend/StartSliceJob.py", line 219, in run
2018-07-06 10:40:14,496 - ERROR - [Thread-3] UM.Logger.logException [88]:     extruders_enabled = {position: stack.isEnabled for position, extruder in stack.extruders.items()}
2018-07-06 10:40:14,496 - ERROR - [Thread-3] UM.Logger.logException [88]:   File "/Users/m.suurmond/Develop/Cura/cura/../plugins/CuraEngineBackend/StartSliceJob.py", line 219, in <dictcomp>
2018-07-06 10:40:14,497 - ERROR - [Thread-3] UM.Logger.logException [88]:     extruders_enabled = {position: stack.isEnabled for position, extruder in stack.extruders.items()}
2018-07-06 10:40:14,497 - ERROR - [Thread-3] UM.Logger.logException [88]: AttributeError: 'GlobalStack' object has no attribute 'isEnabled'